pkg/go/graph/graph.go

```diff
@@ -101,6 +101,23 @@ func (g *AuthorizationModelGraph) Reversed() (*AuthorizationModelGraph, error) {
 	return nil, fmt.Errorf("%w: could not cast to directed graph", ErrBuildingGraph)
 }
 
+// PathExists returns true if both nodes exist and there is a path starting at 'fromLabel' extending to 'toLabel'.
+// If both labels are equal, it returns true.
```

Comment:
https://github.com/gonum/gonum/issues/2014

Reply:
Do we also want to add note here or as FIXME/TODO that for intersection and exclusion, it will not be smart enough to return false if only one side is satisified?

Reply:
I don't think that logic or that TODO belongs here. As far as path traversal goes, our current code is correct. The logic you are talking about is more of business/app logic that belongs in the server, IMO.